app/directives.js

```diff
@@ -0,0 +1,74 @@
+// Directives
+airTrafficControlApp.directive('aircraftDetails', function(){
+   return{
+      templateUrl:'directives/aircraftDetails.html',
+      scope:{
+         aircraftObject: "="
+         // this is causing two way binding, not sure

```

Comment:
I was trying to restrict the scope to only pass in the aircraft object. I have updated the comments. 

// Restrict scope to only access
         // aircraft object
         // This directive will render the aircraft
         // details. In the future, if there was a
         // different way to render the aircraft details,
         // only this directive will have to be changed